module/zfs/zfs_ioctl.c

```diff
@@ -3561,8 +3561,40 @@ zfs_ioc_destroy(zfs_cmd_t *zc)
 
 	if (strchr(zc->zc_name, '@'))
 		err = dsl_destroy_snapshot(zc->zc_name, zc->zc_defer_destroy);
-	else
+	else {
```

Comment:
Lopsided else, the style guide recommends `} else {` for multiline conditionals.